src/components/Calculator.js

```diff
@@ -1,40 +1,51 @@
+import { useState } from 'react';
 import '../App.css';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [result, setResult] = useState({});
+  const handleClick = (e) => {
+    setResult(calculate(result, e.target.textContent));
+  };
+
   return (
     <section>
-      <div className="tableau">0</div>
+      <div className="tableau">{result.next === null ? result.total : result.next}</div>
       <div className="ligne">
-        <span>AC</span>
-        <span>+/-</span>
-        <span>%</span>
-        <span className="orange">+</span>
+        <Span classons="aucun" valeur="AC" cliquons={handleClick} />
+        <Span classons="aucun" valeur="+/-" cliquons={handleClick} />
+        <Span classons="aucun" valeur="%" cliquons={handleClick} />
+        <Span classons="orange" valeur="+" cliquons={handleClick} />
       </div>
```

Comment:
- [ ] Please change this addition symbol because you already have one below and this is supposed to be a division symbol like that `÷` to match the given design you have and because you now can't do division in the calculator.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/229922213-c342ea79-c6b6-410c-8ad6-2f8de256e47b.png)  |![image](https://user-images.githubusercontent.com/68796232/229921678-67cacd1e-af86-4551-a484-9417b4796c71.png)  |